planning/development-strategy.md

```diff
@@ -25,86 +25,161 @@ A simple little website about trees.
 
 <!-- issue title -->
 
-## Site Title
+## Site Title (milestone)
 
 <!-- user story -->
 
-> "as a user I can read the title of the page"
+> "As a user I can read the title of the page"
 
 <!-- detailed description -->
 
-The site needs a clear title that's easy to read at the top of the page.
+<!--The site needs a clear title that's easy to read at the top of the page.-->
```

Comment:
I would leave this line otherwise "detailed description section is empty"

Reply:
yeah..its ok. I missed it. Thanks @MMikhailova